styles/rules.css

```diff
@@ -0,0 +1,31 @@
+.rules {
+  display: grid;
+  place-items: center;
+}
+.rules__container {
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
+  width: fit-content;
+  padding: 2rem;
+  border: 2px solid var(--primary);
+}
+.rules__container ul {
+  list-style: none;
+  padding: 0;
+}
```

Comment:
suggestion - add some space after each ruleset so that they can be clearly distinguished.

Reply:
automatically generated by sass compiler,